packages/analytics-js-common/src/utilities/eventMethodOverloads.ts

```diff
@@ -55,80 +55,87 @@ const pageArgumentsToCallOptions = (
   options?: Nullable<ApiOptions> | ApiCallback,
   callback?: ApiCallback,
 ): PageCallOptions => {
-  const payload: PageCallOptions = {};
+  const payload: PageCallOptions = {
```

Comment:
do the changes in this file pass the sanity suite?

Reply:
Yes.